src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx

```diff
@@ -501,19 +468,20 @@ void CC32xxWiFi::start(WlanRole role, WlanPowerPolicy power_policy)
      */
     wlanRole = role;
     wlanPowerPolicy = power_policy;
-#ifdef SL_API_V2
-    os_thread_create(nullptr, "SimpleLink Task",
-                     OSThread::get_priority_max() - 1, 2048, sl_Task, nullptr);
-#else
-#if 0
-    VStartSimpleLinkSpawnTask(OSThread::get_priority_max() - 1);
-#else
-    xSimpleLinkSpawnQueue = xQueueCreate(3, sizeof( tSimpleLinkSpawnMsg ) );
-    os_thread_create(&xSimpleLinkSpawnTaskHndl, "SimpleLink",
-                     OSThread::get_priority_max() - 1, 2048,
-                     vSimpleLinkSpawnTask, NULL);
-#endif // if 0
-#endif
+
+    /* note, sl_Task MUST be a pthread because the Wi-Fi task uses the pthread
+     * interface such as pthread_self().
+     */
+    pthread_t thread;
+    pthread_attr_t attr;
+    struct sched_param sched_param;
+
+    pthread_attr_init(&attr);
```

Comment:
Yes, TI implemented Pthread support for FreeRTOS in the SDK.  They are 100% compatible with and wrappers on top of FreeRTOS primitives.  This will not tie in with os_thread_create, and will not be a part of our thread list.  The thread lists is kept so we can walk the threads in the Eclipse debugger (check stack sizes, etc...)